packages/site/src/pages/index.tsx

```diff
@@ -305,6 +307,12 @@ const Index = () => {
 
   return (
     <Container>
+      <AlertBanner
+        title={
+          "This is a developer tool for testing purposes only. It shouldn't be used to store any real funds. Use it with caution."
```

Comment:
NitPick: "This is a developer tool for testing purposes only. It should not be used to store any real funds. Use with caution."?

Reply:
[97d6548](https://github.com/MetaMask/snap-simple-keyring/pull/104/commits/97d65489b97f84e2640504405a7731c69b410e49) + [41294a2](https://github.com/MetaMask/snap-simple-keyring/pull/104/commits/41294a20938ae0a1d3a2687b12e47b817a263191)